src/platform-integration/platform-adaptations.md

```diff
@@ -89,14 +89,7 @@ On **iOS**:
 
 ### Platform-specific transition details
 
-On **Android**,
-two page transition animation styles exist depending
-on your OS version:
-
-* Pre API 28 uses a bottom-up animation that
-  [slides up and fades in][].
-* On API 28 and later, the bottom-up animation
-  [slides and clip-reveals up][].
+On **Android**, Flutter uses the [`ZoomPageTransitionsBuilder`][] animation.
```

Comment:
What does that transition do? Can you explain very briefly, like the original text?

Reply:
Done.